### ISSUE TITLE
WCAG 1.4.3: Ensure the contrast between foreground and background colors meets WCAG 2 AA minimum contrast ratio thresholds (.bg-orange-500)

### ISSUE THREAD

**Issue by iamjolly** [uscensusbureau/AI-Toolkit#52]
#### Issue
`Ensure the contrast between foreground and background colors meets WCAG 2 AA minimum contrast ratio thresholds` ([color-contrast](https://accessibilityinsights.io/info-examples/web/color-contrast))
#### Target application
[AI Governance Dashboard](https://uscensusbureau.github.io/AI-Toolkit/pr-preview/pr-34/)
#### Element path
.bg-orange-500
#### Snippet
`<span class="text-sm font-bold px-2 py-1 rounded-full text-white bg-orange-500">20</span>`
#### How to fix
    Fix any of the following:
      Element has insufficient color contrast of 2.82 (foreground color: #ffffff, background color: #ff6900, font size: 10.5pt (14px), font weight: bold). Expected contrast ratio of 4.5:1
#### Environment
Chrome version 140.0.0.0
This accessibility issue was found using Accessibility Insights for Web 2.46.0 (axe-core 4.10.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).
